Library/OcMainLib/OpenCoreKernelPatch.c

```diff
@@ -302,6 +314,16 @@ OcKernelApplyPatches (
       OcKernelApplyQuirk (KernelQuirkAppleXcpmForceBoost, CacheType, DarwinVersion, NULL, &KernelPatcher);
     }
 
+    if (Config->Kernel.Quirks.CustomPciSerialDevice) {
+      RegisterBase = GetSerialRegisterBase ();
```

Comment:
This code executes regardless of `Serial` → `Init`, meaning that config.plist values set in the `Misc` → `Serial` will be ignored here, but this quirk will still be applied. @mhaeuser, is that ok?

Reply:
@vit9696 There already is an ongoing discussion about when we need to init, need to resolve that first: https://github.com/acidanthera/OpenCorePkg/pull/331#issuecomment-1091118957

Reply:
How about applying this patch in OpenCoreMisc.c instead? Is this possible?

Reply:
UPDATE - Resolved by checking `Misc->Serial->Override`.